packages/auth/src/auth-options.ts

```diff
@@ -29,6 +32,11 @@ export const getAuthOptions: GetAuthOptions = (opts) => ({
       clientSecret: opts.azureAdClientSecret,
       tenantId: opts.azureAdTenantId,
     }),
+    // Testing purposes while I don't have Azure setup
+    DiscordProvider({
+      clientId: opts.discordClientId,
+      clientSecret: opts.discordClientSecret,
+    }),
```

Comment:
👁️

Reply:
I never got around to testing the Azure one but can do that first thing tomorrow and remove the temporary discord provider (which btw is a breeze to setup xD)